src/main/java/com/powsybl/openloadflow/dc/equations/DcEquationSystemCreationParameters.java

```diff
@@ -39,6 +39,10 @@ public boolean isIndexTerms() {
         return indexTerms;
     }
 
+    public void setIndexTerms(boolean indexTerms) {
```

Comment:
@geofjamg I am not sure I have to change this parameter this way. Maybe in create dc parameters ?

Reply:
Yes, moved to createDcParameters